CHANGELOG.md

```diff
@@ -14,6 +14,12 @@
   regression from Singularity 2.x. Note that you will now need to use
   `--force` in a build to override a label that already exists in the source
   Docker/OCI container.
+- The source paths for `%files` lines in a definition file are no longer
```

Comment:
@DrDaveD would this be considered a "breaking change" by anybody?

Reply:
It's possible but I doubt it is likely to affect anyone because people shouldn't be expecting to be able to use environment variables in %files, I don't think.  I'm more concerned about the change in globbing behavior that Ian mentioned, and he is looking into an alternative.